### ISSUE TITLE
Concordium client 3.0.4 for Windows is not signed

### ISSUE THREAD

**Issue by mh-concordium** [Concordium/concordium-client#159]
**Bug Description**
Concordium client 3.0.4 for Windows is not signed. Unsigned version can be found on  our official DevDocs page.

**Comment by abizjak** [comment]
I don't think this should be reported as a bug in this repository. It should be either in the developer documentation repo or just reported internally.

There is nothing to do in this repository since both signed and unsigned builds are supported, and the wrong one was linked.
